### PR TITLE
feat: Create New Request e2e test

### PR DESCRIPTION
**Pull request by depapp**
this PR consists of:
- add selector ID
- include an e2e test for the `Create New Request` flow

### CODE REVIEW

File: tests/pages/home.page.js [hunk 1]
```diff
@@ -26,7 +26,12 @@ exports.HomePage = class HomePage {
     // create collection
     this.collectionNameField = page.locator('#collection-name');
     this.submitButton = page.locator(`button[type='submit']`);
-    this.createCollectionToastSuccess = page.getByText('Collection created');
+    this.createNewCollectionSuccessToast = page.getByText('Collection created');
+    this.createNewTab = page.locator('#create-new-tab');
+    this.requestNameField = page.locator('input[name="requestName"]');
+    this.methodName = page.locator('.flex > .StyledWrapper__Wrapper-sc-1tpbhjh-0 > .flex').first();
```
Comment (helloanoop) [inline tests/pages/home.page.js]: @depapp Can we update the selector here ?
I recommend not to use the `StyledWrapper__Wrapper-sc-1tpbhjh-0` auto generated class

**Comment by helloanoop** [comment]
@depapp Thanks for the PR !
I have one comment.

**Comment by depapp** [comment]
> @depapp Thanks for the PR !
> 
> I have one comment.

@helloanoop I don't see any comment from you

**Review by helloanoop — CHANGES_REQUESTED** [review]

File: tests/pages/home.page.js (hunk 1 above)
Comment (depapp) [reply tests/pages/home.page.js]: @helloanoop updated

**Comment by helloanoop** [comment]
Awesome ! Merged.
Thanks @depapp

**Pull request merged**